src/components/Button.tsx

```diff
@@ -7,11 +7,16 @@ type NativeButtonType = ComponentProps<"button">;
 
 type ButtonProps = {
   type?: NativeButtonType["type"];
+  onClick?: NativeButtonType["onClick"];
   children?: ReactNode;
 };
 
-const Button: React.FC<ButtonProps> = ({ type = "button", children }) => (
-  <button type={type} className={style.button}>
+const Button: React.FC<ButtonProps> = ({
+  type = "button",
+  onClick,
+  children,
+}) => (
+  <button type={type} onClick={onClick} className={style.button}>
```

Comment:
And then you wanna take ANY prop that consumer passes in and send it further to the button
```suggestion
const Button: React.FC<ButtonProps> = ({
  type = "button",
  onClick,
  children,
  ...props
}) => (
  <button type={type} onClick={onClick} className={style.button} {...}>
```

Reply:
resolved